src/registrar/tests/test_models_domain.py

```diff
@@ -174,8 +215,12 @@ def test_get_status(self):
         domain.save()
         self.assertIn("ok", domain.status)
 
+    def tearDown(self) -> None:
+        Domain.objects.delete()
+        # User.objects.delete()
 
-class TestRegistrantContacts(TestCase):
+
+class TestRegistrantContacts(MockEppLib):
```

Comment:
remove when branching and we will pull from the setter logic once that one is done.

Reply:
ignore all tests for the same reason